DTXMania/Code/Stage/04.Config/CActConfigList.cs

```diff
@@ -1005,10 +991,10 @@ public void tSetupItemList_Drums()
                 "B: XG仕様9レーン\n" +
                 "C: CLASSIC仕様6レーン",
 
-                "To change the number of lanes.\n" +
-                "10: default 10 lanes\n" +
-                "9: XG style 9 lanes\n" +
-                "6: classic style 6 lanes",
+                "Change the number of drum lanes.\n" +
```

Comment:
I was going to add a note here but wasn't sure if it was correct.

Does changing this setting also automatically merge the hidden lane chips into an available lane? I think it did when I tried the setting.

The Japanese translates roughly to "Change the specifications of the 10-lane score". I think `score` here refers to musical score, which translates to `simfile`.

If that is true, maybe the help text might be better as:

"Change how ten-lane simfile charts are displayed.